spec/visual/samples/bibtex

```diff
@@ -0,0 +1,130 @@
+@article{Witten:1988hf,
```

Comment:
This is just a general comment about this sample but are all of these necessary? I only looked quickly so I perhaps missed the nuance of what was being tested in each one but they all looked very similar. It would be better to have a smaller number that more completely covers the range of possibilities.

Reply:
You are probably right. There is not much happening when the general style is

```
@entry{name,
    key = {value},
```

or `"value"`.

Reply:
Cool. I'll combine all the unique keys into one entry.